.github/workflows/algolia/src/index.ts

```diff
@@ -41,66 +41,29 @@ const getAllFiles = function (
   return arrayOfFiles;
 };
 
-export const checkObjectIds = () => {
-  const objectIds = [];
-
-  const checkObjectId = (matterData: DocsFrontMatter, type) => {
-    if (!matterData.data.objectId) {
-      throw new Error(
-        `objectId missing on file ${type}/${matterData.data.slug}.`
-      );
-    }
-
-    if (objectIds.includes(matterData.data.objectId)) {
-      throw new Error(`objectId ${matterData.data.objectId} appeared twice.`);
-    }
-
-    objectIds.push(matterData.data.objectId);
-  };
-
-  const glossaryFiles = getAllFiles("../../../resources/glossary", [], "glossary");
-  const docsFiles = getAllFiles("../../../resources/docs/simulation", [], "docs");
-
-  const files = [...glossaryFiles, ...docsFiles];
-
-  files.forEach((filePath) => {
-    const file = fs.readFileSync(filePath.inputPath, "utf8");
-
-    const grayMatterData = matter(file) as unknown as DocsFrontMatter;
-
-    checkObjectId(grayMatterData, filePath.type);
-  });
-};
-
 export const generateAlgoliaJson = () => {
-  const objectIds = [];
-
   const getFormattedData = (matterData: DocsFrontMatter, type) => {
-    if (!matterData.data.objectId) {
-      throw new Error(
-        `objectId missing on file ${type}/${matterData.data.slug}.`
-      );
-    }
-
-    if (objectIds.includes(matterData.data.objectId)) {
-      throw new Error(`objectId ${matterData.data.objectId} appeared twice.`);
-    }
-
-    objectIds.push(matterData.data.objectId);
-
     const appendData = {
       ...matterData.data,
       objectId: undefined,
       content: matterData.content,
-      objectID: matterData.data.objectId,
+      objectID: `${type}/${matterData.data.slug}`,
```

Comment:
Links to existing `url_map`s to check paths:
- Docs: https://github.com/hasharchives/hash/blob/main/resources/docs/url_map.json
- Glossary: https://github.com/hasharchives/hash/tree/main/resources/glossary